### PR TITLE
Fix constance settings saving and customize dashboard.

### PR DESCRIPTION
**Pull request by titusz**

### CODE REVIEW

File: iscc_service_generator/admin.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class IsccAdminSite(AdminSite):
+    def get_app_list(self, request):
+        """Customize admin app names and ordering"""
+        applist = super().get_app_list(request)
+
+        ordered = {
+            "iscc_generator": "ISCC Generator",
+            "django_q": "Background Tasks",
+            "auth": "User Managment",
```
Comment (strayer) [inline iscc_service_generator/admin.py:13]:
```suggestion
            "auth": "User Management",
```

File: iscc_service_generator/admin.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            "constance": "Configuration",
+            "admin_interface": "Branding",
+        }
+
+        new_applist = []
+        for app_label, name in ordered.items():
```
Comment (strayer) [inline iscc_service_generator/admin.py:19]: Wouldn’t it make more sense to iterate over applist first and directly look up a pretty name in `ordered` by dict key? Just one loop that way.

**Review by strayer — CHANGES_REQUESTED** [review]

File: iscc_service_generator/admin.py (hunk 2 above)
Comment (titusz) [reply iscc_service_generator/admin.py:19]: Yes but you still need to reorder applist entries to match the order given by ordered :)

Comment (strayer) [reply iscc_service_generator/admin.py:19]: Ah, makes sense.

**Pull request merged**